tests/data/metadata/fence_kubevirt.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" ?>
+<resource-agent name="fence_kubevirt" shortdesc="Fence agent for KubeVirt" >
+<longdesc>fence_kubevirt is an I/O Fencing agent for KubeVirt.</longdesc>
+<vendor-url>https://kubevirt.io/</vendor-url>
+<parameters>
+	<parameter name="action" unique="0" required="1">
+		<getopt mixed="-o, --action=[action]" />
+		<content type="string" default="reboot"  />
+		<shortdesc lang="en">Fencing action</shortdesc>
+	</parameter>
+	<parameter name="plug" unique="0" required="1" obsoletes="port">
+		<getopt mixed="-n, --plug=[id]" />
+		<content type="string"  />
+		<shortdesc lang="en">Physical plug number on device, UUID or identification of machine</shortdesc>
+	</parameter>
+	<parameter name="port" unique="0" required="1" deprecated="1">
+		<getopt mixed="-n, --plug=[id]" />
+		<content type="string"  />
+		<shortdesc lang="en">Physical plug number on device, UUID or identification of machine</shortdesc>
```

Comment:
in the context of this plugin, wouldn't it be more helpful to say `Name of the KubeVirt machine` ?

Reply:
This is a generic description used by all platform